pkg/codegen/report/report.go

```diff
@@ -29,9 +29,11 @@ import (
 	"github.com/pulumi/pulumi/pkg/v3/codegen/hcl2/syntax"
 	hcl2 "github.com/pulumi/pulumi/pkg/v3/codegen/pcl"
 	"github.com/pulumi/pulumi/pkg/v3/version"
+	"github.com/pulumi/pulumi/sdk/v3/go/common/util/env"
 )
 
-const ExportTargetDir = "PULUMI_CODEGEN_REPORT_DIR"
+var ExportTargetDir = env.String("CODEGEN_REPORT_DIR",
```

Comment:
```golang
var ExportTargetDir = env.String("CODEGEN_REPORT_DIR",
```
Move this line into `common/env/env.go`?

Reply:
This environment variable is not actually read by the `pulumi` binary, so I don't want to put it with the rest. This is in the unfortunate part of `pkg/codegen` that is actually a library for downstream consumption.

Because that `codegen/report` is never included, this env doesn't show up in `pulumi env`, which is what we want.

Reply:
Oh, thank makes perfect sense. Brilliant!